saleor/graphql/checkout/mutations.py

```diff
@@ -892,7 +892,7 @@ def perform_mutation(cls, _root, info, checkout_id, store_source, **data):
                     field="checkout_id",
                 )
             except ValidationError as e:
-                checkout_token = from_global_id_strict_type(
+                _type, checkout_token = from_global_id_or_error(
                     checkout_id, Checkout, field="checkout_id"
```

Comment:
inconsistent signature, here we pass Checkout as arg, elswere as `only_type` kwarg.